multiform/forms.py

```diff
@@ -1,14 +1,19 @@
 from __future__ import unicode_literals
 
+from distutils.version import StrictVersion
 from collections import defaultdict, OrderedDict
 from itertools import chain
 from functools import reduce
 
 import operator
 
+from django import get_version as get_django_version
 from django.core.exceptions import ImproperlyConfigured
 from django.forms.forms import BaseForm
-from django.forms.util import ErrorList
+if StrictVersion(get_django_version()) >= StrictVersion('1.9'):
```

Comment:
In Django and other Django related packages I've seen the tendency for this pattern instead.

```python
try:
    from django.forms.utils import ErrorList
except ImportError:
    from django.forms.util import ErrorList
```